cluster-service/Makefile

```diff
@@ -23,6 +24,7 @@ deploy:
 	TENANT_ID=$(shell az account show --query tenantId --output tsv) && \
 	OIDC_BLOB_SERVICE_ENDPOINT=$(shell az storage account show -n ${OIDC_STORAGE_ACCOUNT} -g ${REGIONAL_RESOURCEGROUP} --query primaryEndpoints.blob -o tsv) && \
 	OIDC_ISSUER_BASE_ENDPOINT=$(shell ./oidc-base-endpoint.sh ${OIDC_STORAGE_ACCOUNT} ${REGIONAL_RESOURCEGROUP} ${AFD_OIDC_BASE_ENDPOINT}) && \
+	TLS_CERTIFICATES_ENABLED=true && \
```

Comment:
Why not make enabled configurable, with a default of true ?

Reply:
How would one change it and disable it when deploying a svc/mgmt cluster personal dev environment?

Reply:
We can leave it like this for now and add it in case we need it in the future.

In this case, can we unify the initialization of TLS_CERTIFICATES_ENABLED in a single place in this Makefile? or is there some reason it is initialized in different places?